master/buildbot/master.py

```diff
@@ -132,6 +133,9 @@ def create_child_services(self):
         self.metrics = metrics.MetricLogObserver()
         self.metrics.setServiceParent(self)
 
+        self.stats_service = stats_service.StatsService(self)
```

Comment:
We are extending the concept of 'services' to include more and more configured independant services.
There is no need to have a statsService if it is not configured. c['services'] very well deals with those.